en/_posts/2023-03-22-top-tasks.md

```diff
@@ -0,0 +1,94 @@
+---
+altLangPage: /2023/03/22/les-taches-principales
+css:
+- ../../../css/blog.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+date: 2023-03-22
+dateModified: 2023-03-22
+description: "Top tasks help prioritize what web content needs to be improved. As of 2022, top tasks for Canada.ca are identified using visitors’ responses to the GC Task Success Survey."
+title: "We’re changing the way we identify top tasks for Canada.ca"
+---
+<div class="row">
+  <div class="col-md-8"><img src="/images/thumbs/2023-03-22.png" class="img-responsive" alt="" /></div>
+</div>
+<p class="mrgn-tp-lg">People come to our digital channels to accomplish a very wide range of tasks. We say they’re looking for “information” or “services”, but they don’t make these distinctions. They come to Canada.ca with a goal in mind and a task that they want to complete. If they’re able to accomplish their task, their need has been met. If not, we have work to do.</p>
```

Comment:
Can we change “services”, to “services,”    ...the comma should be inside the quote